lib/capsize/deployer.rb

```diff
@@ -108,6 +108,11 @@ def run_in_isolation
         exit!(0)
       end
 
+      write_out.close
+      read_out.each_line do |line|
+        @deployment.log = (@deployment.log || '') + line
+        @deployment.save!
```

Comment:
This is going to send a *lot* of data across the line, because you're going to be resending the entire log each time. It works for the time being, but I created a ticket to refactor this: https://webonise.atlassian.net/browse/OS-40